src/app/shared/components/message-table/message-table.component.ts

```diff
@@ -9,7 +9,12 @@ import { MainfluxMsg } from 'app/common/interfaces/mainflux.interface';
 })
 export class MessageTableComponent {
   @Input() messages: MainfluxMsg[];
+  @Output() dateEvent: EventEmitter<any> = new EventEmitter();
```

Comment:
You could specify this `any` with `{from: number, to: number}`. (Or `string` if from and to dates are emitted as strings.)